### ISSUE TITLE
Scalatest fails after upgrading SBT

### ISSUE THREAD

**Issue by DavidPerezIngeniero** [sbt/sbt#5169]
## steps
I've upgraded from SBT 1.1.4 to 1.32.  Luckily, few things to change, mainly only `build.properties`.
After that when running ScalaTest tests, I get this error:

```MissingResourceException: Can't find bundle for base name org.scalatest.ScalaTestBundle, locale en_US```

## problem

ScalaTest fails before starting my tests.

## expectation

ScalaTest runs normally

## notes

- Linux 64 bits
- Java 8
- Scala 2.12.10
- I have this: `fork in Test := true`
- related problems:
   - https://stackoverflow.com/questions/21240433/missingresourceexception-cant-find-bundle-for-base-name-com-sun-rowset-rowsetr

**Comment by eed3si9n** [comment]
@DavidPerezIngeniero Thanks for the report. Is `Test / fork := true` not working for you as a workaround?

**Comment by DavidPerezIngeniero** [comment]
Updated Scalatest from 3.04 to 3.08 and the problem has disappeared.

**Comment by michaelahlers** [comment]
For anyone else who comes along, encountered this with SBT 1.3.0, and upgrading to 1.3.3 seems to have cleared this up. Not sure of the root cause.

**Comment by DavidPerezIngeniero** [comment]
I sometimes experience this problem with SBT 1.3.2.
Let's try 1.3.3....

**Comment by DavidPerezIngeniero** [comment]
1.3.3 seems to work better.  :-)

**Comment by michaelahlers** [comment]
Still occurs, typically in CID environments (like Circle CI):

```
java.util.MissingResourceException: Can't find bundle for base name org.scalatest.ScalaTestBundle, locale en
	at java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:1581)
	at java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1396)
	at java.util.ResourceBundle.getBundle(ResourceBundle.java:782)
	at org.scalatest.Resources$.resourceBundle$lzycompute(Resources.scala:8)
	at org.scalatest.Resources$.resourceBundle(Resources.scala:8)
	at org.scalatest.Resources$.makeString(Resources.scala:11)
	at org.scalatest.Resources$.reporterThrew(Resources.scala:304)
	at org.scalatest.CatchReporter.apply(CatchReporter.scala:40)
	at org.scalatest.CatchReporter.apply$(CatchReporter.scala:34)
	at org.scalatest.tools.SuiteSortingReporter.apply(SuiteSortingReporter.scala:26)
	at org.scalatest.tools.Framework$SbtReporter.apply(Framework.scala:1139)
	at org.scalatest.tools.Framework.org$scalatest$tools$Framework$$runSuite(Framework.scala:352)
	at org.scalatest.tools.Framework$ScalaTestTask.execute(Framework.scala:510)
```

And I can't see any common factor affecting it.

**Comment by DavidPerezIngeniero** [comment]
After many times using SBT 1.3.3, this problem has fully disappeared.

**Comment by DJFliX** [comment]
I'm experiencing the issue with SBT 1.3.3 and Scalatest already on 3.0.8. I tried adding `-Dsbt.classloader.close=false` but this unfortunately didn't help. I'm running out of things to try...

**Comment by eed3si9n** [comment]
@DJFliX @michaelahlers Could either of you create a public reproduction project and open a new issue please?

**Comment by DJFliX** [comment]
@eed3si9n I'm trying but the error seems to appear in different kinds of tests (IT, unit) in different modules. So far I've not been able to reproduce it in an isolated test project.

**Comment by pmpfr** [comment]
Unfortunately I can't give a public repro, but I can say we've hit this intermittently on one project (sbt1.3.3 + scalatest 3.0.4) and not on another (same versions; slightly less complicated project).  Not very helpful, sorry.

**Comment by michaelahlers** [comment]
I second @DJFliX and @pmpfr—this problem occurs at random. What makes it harder to reproduce is it seems to happen frequently in a small window, resolves itself after a few retires of the test run in CI, then I don't see it again for a while.

**Comment by dwunsch** [comment]
i can see this with scalatest 3.0.8 and sbt 1.3.4 (never happened with 1.3.3 for me, though)

**Comment by eatkins** [comment]
Is everyone who is experiencing this using `fork = true`? If so, it's hard to understand how the sbt version would impact this because we haven't changed anything about fork recently. Moreover, there are no classloader tricks used in fork: we simply set the classpath of the forked java process to be the full classpath of the project + the sbt test agent jar.

The only thing that I can think of is that the particular resource is found in a scalatest jar which is in the coursier cache. I guess it's possible that the jar gets overwritten during the forked run? That would potentially be a coursier bug and the coursier version has changed between the various 1.3.x releases. This is complete speculation however.

**Comment by eatkins** [comment]
Actually based on #5262, I doubt it is a coursier issue.

**Comment by oriarditi** [comment]
Still experiencing this sporadically with `fork = true` and SBT 1.3.4

**Comment by DavidPerezIngeniero** [comment]
I would try the latest SBT 1.3.5

**Comment by KadekM** [comment]
Sporadically experiencing with SBT 1.3.3, haven't tried latest yet
